addons/binding/org.openhab.binding.oceanic/build.properties

```diff
@@ -3,4 +3,4 @@ output.. = target/classes
 bin.includes = META-INF/,\
                .,\
                OSGI-INF/,\
-               ESH-INF/
\ No newline at end of file
+               ESH-INF/           
```

Comment:
The "return" key does not work...?
You could actually simply remove this file from your changeset, that would be the simplest solution!